### PR TITLE
Extend Readme on how-to run tutorial locally

### LINKED ISSUE #58: Readme: update section on "run the tutorial on your computer"
We should add a few lines explaining how to download the archive with the tutorial

### PR DESCRIPTION
**Pull request by edoardob90**
Fixes #58

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,23 +5,52 @@
...
 ```
 
+Alternatively, you can manually download a ZIP archive with the latest version of the material:
+
+[Download ZIP archive](https://github.com/empa-scientific-it/python-tutorial/archive/refs/heads/main.zip)
+
+Extract the archive in a directory of your choice.
+
+### 2. Create a dedicated environment
+
```
Comment (yakutovicha) [inline README.md:29]: Maybe add something like:
```suggestion

Enter the tutorial folder:
```console
$ cd path/to/python-tutorial
```
```

File: README.md [hunk 2]
```diff
@@ -5,23 +5,52 @@
...
 ```console
 conda activate python-tutorial
 ```
 
-Finally, launch JupyterLab with
+You can update the existing environment (that is, downloading the latest version of the packages) with
+
 ```console
-jupyter lab
+conda env update -f environment.yml
```
Comment (yakutovicha) [inline README.md:48]:
```suggestion
conda env update -f binder/environment.yml
```

**Review by yakutovicha — CHANGES_REQUESTED** [review]
A few small changes.

File: README.md (hunk 1 above)
Comment (edoardob90) [reply README.md:29]: I'll apply this in another commit: it's not clear if the `suggestion` is  messing up with the code block

Comment (edoardob90) [reply README.md:29]: I'll apply this in another commit: it's not clear if the `suggestion` is  messing up with the code block

**Review by yakutovicha — APPROVED** [review]

**Pull request merged**